src/config.c

```diff
@@ -17,8 +17,10 @@
 #include "specific/s_shell.h"
 #include "util.h"
 
+#include <stdio.h>
 #include <string.h>
 
+#define CONTROL_LAYOUT_FMT "layout_%d"
```

Comment:
I think this define is a bit overzealous since we don't reuse other hardcoded setting names either between the read/write routines.